modules/videoio/misc/python/pyopencv_videoio.hpp

```diff
@@ -1,6 +1,8 @@
 #ifdef HAVE_OPENCV_VIDEOIO
+#include "../../src/cap_interface.hpp"
```

Comment:
Is that ok to have relative import path? I don't want to move `CvStream` to public API

Reply:
This approach might not work with standalone python bindings build or some other special configuration.

As an alternative you can make it "technically" public by moving class declaration to a `.private.hpp` header like we do with [avi container](https://github.com/opencv/opencv/blob/4.x/modules/videoio/include/opencv2/videoio/container_avi.private.hpp) - symbols will be exported, but header will not be installed.

Reply:
Thanks. If that's not critical, I moved to `utils.private.hpp`. We can avoid this by small duplication of the struct `CvStream : public std::streambuf` but in bindings.